Source: https://bugzilla.redhat.com/show_bug.cgi?id=1361157
Upstream: Reported, https://bugs.chromium.org/p/chromium/issues/detail?id=604652
Reason: Fix crashing tabs and extensions when chromium is build against glibc 2.24

--- a/third_party/WebKit/Source/wtf/allocator/PageAllocator.cpp	2016-08-03 21:02:37.000000000 +0200
+++ b/third_party/WebKit/Source/wtf/allocator/PageAllocator.cpp	2016-08-14 10:59:51.395354850 +0200
@@ -41,6 +41,11 @@
 #include <errno.h>
 #include <sys/mman.h>
 
+#if OS(LINUX) && defined(MADV_FREE)
+// Added in Linux 4.5, but we don't want to depend on 4.5 at runtime
+#undef MADV_FREE
+#endif
+
 #ifndef MADV_FREE
 #define MADV_FREE MADV_DONTNEED
 #endif
